### ISSUE TITLE
Stopped working. Can't revert to stock settings.

### ISSUE THREAD

**Issue by arniep** [valinet/ExplorerPatcher#2822]
Just installed the latest version on EP but this app has really messed up my W11 pro HP computer. When I first installed it yesterday, it seemed to work as advertised. However, when I went to use it today, it was terrible... I set it up to imitate W10 file explorer and start menu and initially it seemed to work great yesterday. When I went to use it today, without having changed anyting on my PC, left-clicking on the start button caused a few seconds on the spinning wheel and then nothing. No start menu appeared. Right-clicking on the Start button did open the usual utilities menu, but clicking on Settings was problematic. It did open the Settings menu but clicking on any of the settings categories dumped immediately back to the desktop. The only item that worked was the shutdown/restart button. 
So I thought I would try to uninstall ep_setup.exe. I found a note on their github to rename the app to ep_uninstall.exe which I did. That gave me a success message and I then rebooted. ALthough I now have the W11 start menu button behavior back, I still cannot open anything within Settings without still getting dumped right back to the desktop. Apparently, Explorer Patcher even changed my trackpad to click only, no tapping and cannot even get back into the trackpad/mouse settings now.
Help!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

You may also need to repair your windows install.  Please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by Amrsatrio** [comment]
A very weird issue that I have no idea what could cause it... I've had no problems with ExplorerPatcher since 2021....

**Comment by arniep** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/3038125/ed7bb128-3702-4d08-895e-dc25c968dde4)
I hope this screenshot worked. 

Here is the EP version I installed and tried to remove:
22621.2861.62.2

I don;t have anything else that I can think of that would modify the Windows UI.

Here is a screenshot of the chkdsk, which seems to show no errors:

![image](https://github.com/valinet/ExplorerPatcher/assets/3038125/b8d1bdf1-a4bb-4934-9b86-4d579bac1aa7)

What do you suggest, with this information?
Any sub-category in Windows settings that I click on just immediately dumps me back to my desktop, so I cannot change any Windows settings!
Thank you.

**Comment by arniep** [comment]
Do you think I should try to re-install EP?

**Comment by Amrsatrio** [comment]
Try to go to EP Properties > File Explorer and uncheck Register as shell extension.

**Comment by pyrates999** [comment]
Please follow the complete instructions for repairing your windows install.  You only ran chkdsk it seems.

**Comment by arniep** [comment]
I ran all those commands, then rebooted, as you suggested. No improvement. Still acting the same. Any other ideas to get me back to where I can access Windows Settings submenus?

Here is a screenshot of the commands I ran that I have attached.
Do you think I should try to install EP again? 
It did work properly for about the first day after I installed it. The day after that is when I started noticing the issues even though I had not made any changes to EP or anything else on my W11 machine.
![EP_Stopped working  Can't revert to stock settings  · Issue #2822 · valinet_Explore](https://github.com/valinet/ExplorerPatcher/assets/3038125/e3eb8a32-d602-4caa-8327-27bba0933e7f)

**Comment by pyrates999** [comment]
run **sfc /scannow** until it no longer shows any corrupt files.

**Comment by arniep** [comment]
First, I want to thank you for taking your time to try to help me with this.
I did run the sfc /scannow command and got a 'no errors' message (screenshot attached)
Unfortunately, upon reboot nothing had changed. The problem persists! What to do now?
Thank you again.
![EP_2024-02-15 16_00_19-Stopped working  Can't revert to stock settings  · Issue #2822 · valinet_Explore](https://github.com/valinet/ExplorerPatcher/assets/3038125/da3cba9e-604a-4eeb-89f8-689646712ca7)

**Comment by pyrates999** [comment]
Try this then: https://www.computerworld.com/article/3262979/how-to-fix-windows-10-with-an-in-place-upgrade-install.html

**Comment by arniep** [comment]
That worked! I can now get into Windows settings but I dare say I am little loath to give EP another try. I did notice that while my computer does seem to be acting normally, I still cannot get into the settings for my Logitech T650 trackpad. I tried reinstalling their software, but it closes immediately after I try to open it. I will work on the some more tomorrow, but at least everything else seems to be working and the in-place re-install of Windows11 went smoothly.
Thank you again for your time and help!
